ckb-debugger-tests/src/bin/gr-update.rs

```diff
@@ -40,7 +53,14 @@ pub fn main() -> Result<(), Box<dyn std::error::Error>> {
     output_data.extend_from_slice(child_script_config.as_slice());
     repr_tx.tx.outputs_data[0] = JsonBytes::from_vec(output_data);
 
-    let witness_args = create_witness_args(&child_script_config, 0, &[Bytes::default()])?;
+    let combine_lock_witness =
```

Comment:
simplify it with "create_lock_wrapper_witness".

Reply:
There is no need to pass `wrapped_script`